### PR TITLE
Feature/programs where it started

### PR DESCRIPTION
**Pull request by bradybridges**
Adds where it started section to the programs page. This completes the programs page. There are still a few components to go over to see if the content for them is in contentful. ***These components can be referenced in issue 36.*** The commits for this branch should clean up after we merge the other active PRs, feel free to hold off on reviewing this until those are merged.

image

### CODE REVIEW

File: src/components/layout.scss [hunk 1]
```diff
@@ -4,6 +4,10 @@
   flex-wrap: wrap;
 }
 
+.programs .content-container {
+  padding: 100px 84px 0;
+}
+
```
Comment (LayaTaal) [inline src/components/layout.scss]: What is this for? I'm not sure it is doing anything.

File: src/components/programResources.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  p {
+    font-size: var(--headerFontSizeTertiary);
+    line-height: var(--headerLineHeightSecondary);
+    font-family: var(--scopeOne);
+    margin: 0;
+  }
+}
+
+.program-resources__button {
```
Comment (LayaTaal) [inline src/components/programResources.scss]: This button and the button in the section above (as well as the homepage) share this same button style. Let's go ahead and make this a button component with some standard styles.

File: src/components/header.scss [hunk 3]
```diff
@@ -4,7 +4,7 @@
   position: fixed;
   display: flex;
   justify-content: center;
-  z-index: 10;
+  z-index: 100;
```
Comment (LayaTaal) [inline src/components/header.scss]: Is this necessary? Not sure if you are addressing something in this section or a bug elsewhere.

**Review by LayaTaal — CHANGES_REQUESTED** [review]
Looks good! Just a few small requests.

File: src/components/layout.scss (hunk 1 above)
Comment (bradybridges) [reply src/components/layout.scss]: @LayaTaal That one is just removing the padding on the bottom of the content container on the programs page. I'll update that the set the padding-bottom so it's clearer what it's doing.

File: src/components/header.scss (hunk 3 above)
Comment (bradybridges) [reply src/components/header.scss]: I must have put that in a little early, I was having an issue with something overlapping the header. I'm not able to find it anymore though.

**Review by LayaTaal — APPROVED** [review]
Thanks for moving that button into a component. Merge away!

**Pull request merged**